python/dolfinx/la/petsc.py

```diff
@@ -29,6 +29,15 @@
 __all__ = ["assign", "create_vector", "create_vector_wrap"]
 
 
+def ghostUpdate(x: PETSc.Vec, insert_mode: PETSc.InsertMode, scatter_mode: PETSc.ScatterMode):  # type: ignore
```

Comment:
I do understand where the name is coming from, but for consistency with the rest of dolfinx codebase maybe we want to call it `ghost_update`?

Reply:
I think this function should be 'private'. We aim to minimise the wrapping of PETSc functionality and prefer users to use the petsc4py interfaces.